day_10/task_2/tests/pipe_maze_tests.cpp

```diff
@@ -221,14 +221,23 @@ TEST(AttributedMazeTest, GivenAttributedMazeWithMarkedLoop_WhenCalculateEnclosed
         ".|..|.|..|.",
         ".L--J.L--J.",
         "...........",
-    }
+    };
     AttributedMaze maze(transform_to_attributed_maze(pipe_maze));
     maze.mark_loop_tiles_in_attributed_maze();
 
-
     // When
     const auto result = maze.count_enclosed_tiles();
 
     // Then
     EXPECT_EQ(result, 4);
 }
+
+//  PipeMaze pipe_maze = {
+//         "L--JL--J",
+//         "|..||..|",
+//         "|L-7F-J|",
+//         "||....||",
+//         "||....||",
+//         "|F----7|",
+//         "S------7",
+//     };
```

Comment:
Please remove it or transform it into the unit test.

Reply:
new test added